AccountingSystem/Client/DataStoreClient.py

```diff
@@ -137,14 +137,18 @@ def ping( self ):
 def _sendToFailover( rpcStub ):
   """ Create a ForwardDISET operation for failover
   """
-  request = Request()
-  request.RequestName = "Accounting.DataStore.%s.%s" % ( time.time(), random.random() )
-  forwardDISETOp = Operation()
-  forwardDISETOp.Type = "ForwardDISET"
-  forwardDISETOp.Arguments = DEncode.encode( rpcStub )
-  request.addOperation( forwardDISETOp )
-
-  return ReqClient().putRequest( request )
-
+  try:
+    request = Request()
+    request.RequestName = "Accounting.DataStore.%s.%s" % ( time.time(), random.random() )
+    forwardDISETOp = Operation()
+    forwardDISETOp.Type = "ForwardDISET"
+    forwardDISETOp.Arguments = DEncode.encode( rpcStub )
+    request.addOperation( forwardDISETOp )
+
+    return ReqClient().putRequest( request )
+
+  # We catch all the exceptions, because it should never crash
+  except Exception as e:  # pylint: disable=broad-except
+    return S_ERROR( "Exception sending accounting failover request: %s" % repr( e ) )
```

Comment:
I thought of it, but I don't know what to put in there when we catch a generic exception. 
I can put the generic place holder for the RMS error ?